docs/developer/actors/update.mdx

```diff
@@ -1,16 +1,13 @@
 ---
-title: "Customizing the actor"
+title: "Iterate"
 date: 2018-12-29T09:00:00+00:00
-sidebar_position: 6
+sidebar_position: 3
 draft: false
 ---
 
-import Tabs from "@theme/Tabs";
-import TabItem from "@theme/TabItem";
-
 This section describes how you can iteratively build and test your actor functionality.
 
-## `wash dev`
+## wash dev
 
 As of wash 0.18.0 you can use the experimental `wash dev` command to rapidly build and iterate on
```

Comment:
Side note: IMO if we're going to rely on `wash dev` in our documentation we should remove the experimental flag from it

Reply:
For sure. I think wash dev should get support for wadm manifests and then we drop the experimental flag

Reply:
I'll resolve this for now, and we can come back to this after https://github.com/wasmCloud/wasmCloud/pull/1269 is released